package.json

```diff
@@ -77,7 +79,9 @@
     "nock": "13.0.4",
     "rimraf": "3.0.2",
     "semantic-release": "19.0.3",
-    "semantic-release-slack-bot": "3.5.2"
+    "semantic-release-slack-bot": "3.5.2",
+    "ts-jest": "^27.1.2",
```

Comment:
Not sure if that's relevent here, but on the server we usually use `swc/jest` instead of `ts-jest`

Reply:
Done thanks to @Thenkei